chart/kubeapps/templates/apprepository-rbac.yaml

```diff
@@ -91,8 +91,7 @@ rules:
 # The Kubeapps app repository controller can read and watch its own
 # AppRepository resources cluster-wide. The read and write cluster-roles can
 # also be bound to users in specific namespaces as required.
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
```

Comment:
have you removed the `kind` by accident?

Reply:
this seems why the CI is failing:

```
Error: UPGRADE FAILED: error validating "": error validating data: kind not set
```

Reply:
Correct. I'll fix it